### DISCUSSION TITLE
I AM LOCKED OUT MY ACCOUNT OF 15 YEARS ... PLZ HELP SOME ONE ... !!! SOS!!

### DISCUSSION THREAD

**Discussion by keyabibo** [community/community#124599] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Everything was great all these years ... just recently I got a new iPhone so I added the Google Authenticator app as I had on my old one but I had to rescan the QR code from github ... things where fine. I was logged in via Chrome and was trying to log in via the Opera browser and that's when it started failing to authenticate the 2FA digits ... so I used one of my emergency numbers to get in and rescanned the QR code and tried again ... but nothing worked so far ... I've started a recovery process which completely automated and I have no idea what the heck to do at this point ... to even get to the support portal I needed to log in ... what the heck is this ... no email for support works ... we're still humans and we have situations that automated bots won't understand ... in the entire recovery process there wasn't one place that I could say what my problem is ... No I've just lot access to about 20 years of my work ... can anyone pls help ... is there anyway to get to a human being ... at this rate, even if they help me recover the account I might be having the same issue again and again ... where am I gonna go then? who can I talk to then ... can anyone pleeeeease help ... I sware to God, after this experience I'm getting out of this stupid system where human being don't mean anything at all ... so can anyone plz help ....

any help in the right direction will be highly appreciated....

![image (1)](https://github.com/community/community/assets/170224880/ec6e920e-9609-4d3c-95a4-a8284ba72ad6)
![image](https://github.com/community/community/assets/170224880/87614bdb-d8b2-448f-b3b8-403f1a101b3f)

**Comment by ghost** [comment]
GitHub Support Links
[GitHub Password Reset](https://github.com/password_reset)
[GitHub Account Recovery Form](https://support.github.com/request/account_recovery)
[GitHub Support Contact](https://support.github.com/contact)

https://support.github.com/request

**Comment by CopperEagle** [comment]
Hey @keyabibo

Good thing you post this here, GitHub staff will see this eventually. Note that in the past, it has still taken them a while to answer... just be patient. AFAIK, there are few other ways to contact staff these days, unfortunately. (For free accounts anyways.)

So since you started the account recovery process, keep checking your email. There should be a link arriving in a few days that allows to resetup the 2FA. If they deny, you get further contact. In the meantime, read [this section](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication) about email recovery. Just **don't** unlink your email address.

Right now, probably the best thing is to check the email and be patient, I'm afraid.

**Comment by karaposu** [comment]
Sounds horrible. What is the point of backup codes if they wont work in such cases?
